pynicotine/portmapper.py

```diff
@@ -51,8 +51,9 @@ class NATPMP(BaseImplementation):
 
     NAME = "NAT-PMP"
     REQUEST_PORT = 5351
-    REQUEST_TIMEOUT = 1  # 1 second
     SUCCESS_RESULT = 0
+    REQUEST_ATTEMPTS = 2 # spec says 9, but 2 should be enough
+    REQUEST_INIT_TIMEOUT = 0.250 # seconds
```

Comment:
Still a couple of linting errors to resolve:
https://github.com/nicotine-plus/nicotine-plus/actions/runs/4269222146/jobs/7432291890#step:5:12